plugins/metrics/plugins/metrics/html/metrics-response.html

```diff
@@ -59,4 +59,61 @@
 
 
 </div>
+<div ng-controller="QuickAlertController as qac">
+  <div ng-if="!showQuickAlert" class="col-sm-9 col-md-10 content">
```

Comment:
I think you really want to use ng-show/ng-hide instead of ng-if because they just show/hide elements where as ng-if physically adds/removes those elements from DOM tree. This can have performance impacts.